### ISSUE TITLE
No default key could be found for new kindle app [mac]

### ISSUE THREAD

**Issue by pcpLiu** [noDRM/DeDRM_tools#432]
### Question / bug report

![image](https://github.com/noDRM/DeDRM_tools/assets/3357328/797ef724-8ed7-4eee-bcff-21f7b146c8a3)
Hey, Seems i cannot find the default key when Im using the latest kindle Mac app (Version: 7.0.0.100 (1.316222)
).

Tried to use v1.7 but it's too old for the system.

Anyone know in which folder I can find the key>

### Which version of Calibre are you running?

6.29

### Which version of the DeDRM plugin are you running?

10.0.3

### If applicable, which version of the Kindle software are you running?

7.0.0.100 (1.316222)

### Log output

```log
The default encryption key for Kindle for Mac/PC could not be found
```

**Comment by ZolaLa9** [comment]
You need to be using [Kindle Classic app v 1.40.x](https://apps.apple.com/us/app/kindle-classic/id405399194?mt=12) available in the Mac App Store and [DeDRM 10.0.9](https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.9)

The new Kindle for Mac app 7.0 will not work with DeDRM

**Comment by pcpLiu** [comment]
ah...thanks!!!
